aten/src/ATen/native/native_functions.yaml

```diff
@@ -5912,17 +5912,22 @@
     CPU: _solve_helper_cpu
     CUDA: _solve_helper_cuda
 
-- func: cholesky_inverse.out(Tensor self, bool upper=False, *, Tensor(a!) out) -> Tensor(a!)
-  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
+- func: _cholesky_inverse_out_helper_(Tensor(a!) self, Tensor(b!) infos, bool upper) -> Tensor(a!)
```

Comment:
@ngimel thinks we can get rid of this native_functions.yaml entry, since this function only appears to be used once internally. See https://github.com/pytorch/pytorch/pull/43215, where Kurt used DEFINE_DISPATCH and DECLARE_DISPATCH macros to get rid of a native_functions.yaml entry.